src/collection-view.js

```diff
@@ -295,7 +296,20 @@ Marionette.CollectionView = Marionette.AbstractView.extend({
       });
     }
 
-    return childView;
```

Comment:
So the issue in question was https://github.com/marionettejs/backbone.marionette/issues/1861 with passing collection as an array or a backbone.collection.  Rather than solve it, the revert also considered that it might not be a super useful feature on its own.

But in this edge case, really the user should not have to override `_getChildView`, but in the case that there are multiple instances of Backbone, simply always make `childView` a function that returns the class rather than passing the class directly... simple enough.

It might be useful to have some reference on the error though?  So that someone can immediately determine what is wrong, as this edge case might be a major headache to debug otherwise.

@jasonLaster thoughts?